### PR TITLE
DEV-1635: Update Drop-ins version to v2.2.2

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

File: pages/docs/drop-in-components/index.mdx [hunk 1]
```diff
@@ -994,9 +994,18 @@ Get started building with drop-in components by checking out the [getting starte
...
 ## Changelog
 
-### Latest `v2.2.1`
+### `v2.2.2` (Latest)
 
-The latest version of **dwolla-web.js** is `2.2.1`. If you are currently using `v2`, we recommend [upgrading](https://developers.dwolla.com/docs/drop-in-components/drop-ins/setup-and-configuration#include-dwolla-webjs) to `2.2.1`.
+The latest version of **dwolla-web.js** is `2.2.2`. If you are currently using `v2`, we recommend [upgrading](https://developers.dwolla.com/docs/drop-in-components/drop-ins/setup-and-configuration#include-dwolla-webjs) to `2.2.2`.
+
+- Updated user hints and prompts for documentation upload screens to be correct and consistent throughout the following drop-in components:
+  - `<dwolla-account-opening>`
```
Comment (spencerhunter) [inline pages/docs/drop-in-components/index.mdx]: @ShreyaThapa Is this a new component?

Comment (ShreyaThapa) [reply pages/docs/drop-in-components/index.mdx]: Not a new component, but I just realized we don't document it. I think I'll just remove its mention.

Comment (spencerhunter) [reply pages/docs/drop-in-components/index.mdx]: Ah, didn't know that! Sounds good👍

Comment (ShreyaThapa) [reply pages/docs/drop-in-components/index.mdx]: Looks like we only mention it in the changelog - https://developers.dwolla.com/docs/drop-in-components#v219

Comment (ShreyaThapa) [inline pages/docs/drop-in-components/index.mdx]:
```suggestion
```

**Review by spencerhunter — APPROVED** [review]

**Pull request merged**